server/index.js

```diff
@@ -10,7 +10,33 @@ app.use(bp.urlencoded({extended: true}));
 app.use(bp.json());
 
 app.get('/propertyDetails/', (req, res)=>{
-  console.log('do something');
+  let val = Object.values(req.query);
```

Comment:
@pauly-van something to verify: whether we are supposed to set up multiple request endpoints in the server, or if the server is supposed to send a single request to the db which makes multiple queries, packages them into one response, and sends that back to the server to send back to the client (I had thought it was the latter)